control-plane-operator/controllers/hostedcontrolplane/csi/kubevirt/files/controller.yaml

```diff
@@ -0,0 +1,109 @@
+# sourced from https://github.com/kubevirt/csi-driver/tree/main/deploy/split-infra-tenant
+kind: Deployment
+apiVersion: apps/v1
+metadata:
+  name: kubevirt-csi-controller
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: kubevirt-csi-driver
+  template:
+    metadata:
+      labels:
+        app: kubevirt-csi-driver
+    spec:
+      serviceAccount: kubevirt-csi
+      priorityClassName: hypershift-control-plane
+      containers:
+        - name: csi-driver
+          imagePullPolicy: Always
+          image: quay.io/dvossel/kubevirt-csi-driver:latest
```

Comment:
can we please update this image?

Reply:
fyi for anyone reviewing this. the images are all overwritten in these yaml files using the release payload